src/components/Organization/Invite.tsx

```diff
@@ -0,0 +1,252 @@
+import {
+  Alert,
+  Box,
+  Button,
+  ButtonProps,
+  Flex,
+  FlexProps,
+  FormControl,
+  FormLabel,
+  Heading,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Progress,
+  Radio,
+  Stack,
+  Text,
+  useDisclosure,
+  useRadio,
+  useToast,
+} from '@chakra-ui/react'
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query'
+import { useOrganization } from '@vocdoni/react-providers'
+import { ensure0x } from '@vocdoni/sdk'
+import { ReactElement } from 'react'
+import { FormProvider, useController, useForm, useFormContext } from 'react-hook-form'
+import { Trans, useTranslation } from 'react-i18next'
+import { ApiEndpoints } from '~components/Auth/api'
+import { HSeparator } from '~components/Auth/SignIn'
+import { useAuth } from '~components/Auth/useAuth'
+import InputBasic from '~components/Layout/InputBasic'
+import { CallbackProvider, useCallbackContext } from '~utils/callback-provider'
+
+type InviteData = {
+  email: string
+  role: string
+}
+
+type Role = {
+  role: string
+  name: string
+  writePermission: boolean
+}
+
+// Hook to fetch roles
+const useRoles = () => {
+  const { bearedFetch } = useAuth()
+
+  return useQuery({
+    queryKey: ['organization', 'roles'],
+    queryFn: async () => {
+      const response = await bearedFetch<{ roles: Role[] }>(ApiEndpoints.OrganizationsRoles)
+      return response.roles
+    },
+    staleTime: 60 * 60 * 1000,
+    select: (data) => data.sort((a, b) => a.name.localeCompare(b.name)),
+  })
+}
+
+// Hook to handle member invitation mutation
+const useInviteMemberMutation = () => {
+  const { bearedFetch } = useAuth()
+  const { organization } = useOrganization()
+  const queryClient = useQueryClient()
+
+  return useMutation({
+    mutationFn: async (body: InviteData) =>
+      await bearedFetch(ApiEndpoints.OrganizationMembers.replace('{address}', ensure0x(organization.address)), {
```

Comment:
During this call the button of invitation stays enable and no loader is being showed that could lead to the user clicking multiple times

Reply:
fixed [`0195ae5` (#828)](https://github.com/vocdoni/ui-scaffold/pull/828/commits/0195ae532fbcb45f1330697c3d1f0a12516c1a3c)